openvpn/mbedtls/ssl/sslctx.hpp

```diff
@@ -135,7 +135,12 @@ namespace openvpn {
 
 	  MBEDTLS_TLS_ECDH_ECDSA_WITH_3DES_EDE_CBC_SHA,
 
-	  0
+	  /* Selected CHACHA20-POLY1305 ephemeral suites */
+	  MBEDTLS_TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,
+      MBEDTLS_TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,
```

Comment:
Whitespace error

Reply:
whitespace fixed

Reply:
This now looks even more wrong. The idea was not to change all old/other lines to the new indentation that you use but rather have your new line conform with the whitespace that is already there